doc/release/v2_3_70_2.md

```diff
@@ -9,6 +9,8 @@ Bug Fixes
 ---------
 
 * Fixed several warnings reported by clang static analyzer.
+* Fixed warnings reported by coverity static analyzer on Random class, for 
+  now replacing risky functions with functions from c++11 random library.
```

Comment:
`Random` --> `yarp::os::Random` class . 

It is not clear why you write "for now" or "replacing risky functions" : this are release notes, so just state what is changing for the users, something like: 
* Migrated `yarp::os::Random` from `cstdlib`'s functions `srand` and `rand` to  C++11 `std::default_random_engine`. Consequently calling `yarp::os::Random` methods will not influence anymore the global state of the `srand` and `rand` functions.

Reply:
You're right thanks, I will edit this